config/routes.rb

```diff
@@ -14,4 +14,8 @@
   resources :charts, only: %i[show index create]
   # get 'charts/show'
   get 'dashboard/show'
+  resources :users
+  get 'invitations/update'
+  get 'invitations/create'
```

Comment:
It will be work? I suppose that create/update/destroy it is not a get

Reply:
Yes, you are right. I fixed it.